### DISCUSSION TITLE
Multiple scenes simultanously

### DISCUSSION THREAD

**Discussion by PtrBld** [enable3d/enable3d#320] [Help / Questions] [answered]
Hi, I am currently using a normal Phaser Scene together with a Scene3D (shown simultanously) and it works great. 
However, now I try to use two 3d enabled scenes simultanously on top of each other and only one is shown. I guess it has something to do with the background of the scenes not being transparent but I cannot get it to work. 

```
this.game.scene.start('BackgroundScene')
this.game.scene.start('ForegroundScene')
```

I have already set the alpha to true in the third dimension config of the foreground scene:

```
this.accessThirdDimension({
  alpha:true,
  usePhysics: false,
  ...
});
```

Is this even possible atm? Since it works with Phaser 2d scenes I thought it should also work with the 3d scenes?

**Comment by yandeu** [comment, marked as answer]
Unfortunately, you can't easily run multiple enable3d scene on top of each other.
